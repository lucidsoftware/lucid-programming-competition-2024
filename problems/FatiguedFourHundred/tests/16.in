version https://git-lfs.github.com/spec/v1
oid sha256:14a3b3e0aa76ba62e3312c6bb2d98beb2619c175f0e1faa00bf25b387eef3d34
size 482307
