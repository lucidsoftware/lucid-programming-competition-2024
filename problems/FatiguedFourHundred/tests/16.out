version https://git-lfs.github.com/spec/v1
oid sha256:57188e0a284462b9a4ef84f8cddc219251caf0704774babf27031cc4d619c41d
size 356417
